plugin/ensime.vim

```diff
@@ -38,6 +37,10 @@ command! -nargs=* -range EnToggleFullType call ensime#com_en_toggle_fulltype([<f
 command! -nargs=* -range EnOrganizeImports call ensime#com_en_organize_imports([<f-args>], '')
 command! -nargs=* -range EnAddImport call ensime#com_en_add_import([<f-args>], '')
 
+function! EnPackageDecl() abort
+  return ensime#com_en_package_decl()
+endfunction
```

Comment:
> Regarding the issue that you point out about autoload and the wrappers, could you please elaborate on what you mean and what are you proposing? Good ideas are welcome 👍 

Basically I meant, do we need the `function! EnPackageDecl()` at all? All that it does is call the `ensime#fun_en_package_decl()` function, so couldn't we use the latter function directly and not have two pieces of code to maintain? Unless they are script-scoped with `s:`, all of the functions are "public API" already, they can be called from user code/config. It's best for us to think of them this way when designing the plugin IMO, because it is a fact we can't avoid 😃 

We don't have a ton of functions on the VimL side yet except for those that dispatch for commands, since substantial implementation is on the Python side, but I'm sure the Vim API will grow over time as users want it.